facebook/OpenBIC
# YV4: sd: Remove ina233 sensors on T1C system

# Description:
- Since there is no E1S data drive on the T1C system, remove the ina233 sensors monitoring the E1S data drive.

# Motivation:
- Since there is no E1S data drive on the T1C system, remove the ina233 sensors monitoring the E1S data drive.

# Test Plan:
- Build code: Pass
- Check sensors on T1C system: Pass

# Test Log:
- Check sensors on T1C system root@bmc:~# busctl tree xyz.openbmc_project.PLDM | grep Sentinel_Dome_Slot_1_ | wc -l 62
root@bmc:~# busctl tree xyz.openbmc_project.PLDM | grep Sentinel_Dome_Slot_1_
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_INA233_E1S_BOOT_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_VR_CPU0_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_VR_CPU1_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_VR_PVDD11_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_VR_PVDDIO_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/current/Sentinel_Dome_Slot_1_MB_VR_SOC_CURR_A
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_CPU_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_A_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_B_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_C_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_D_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_E_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_G_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_H_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_I_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_J_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_DIMM_K_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_INA233_E1S_BOOT_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_VR_CPU0_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_VR_CPU1_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_VR_PVDD11_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_VR_PVDDIO_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/power/Sentinel_Dome_Slot_1_MB_VR_SOC_PWR_W
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_CPU_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_A_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_B_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_C_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_D_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_E_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_G_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_H_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_I_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_J_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_DIMM_K_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_FIO_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_INLET_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_OUTLET_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_SSD_BOOT_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_VR_CPU0_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_VR_CPU1_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_VR_PVDD11_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_VR_PVDDIO_TEMP_C
    │ │ ├─ /xyz/openbmc_project/sensors/temperature/Sentinel_Dome_Slot_1_MB_VR_SOC_TEMP_C
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P12V_DIMM_0_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P12V_DIMM_1_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P12V_STBY_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P1V2_STBY_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P1V8_STBY_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P3V3_STBY_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P3V_BAT_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_P5V_STBY_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_PVDD11_S3_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_PVDD18_S5_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_PVDD33_S5_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_SIDECAR_DETECT_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_ADC_SLOT_DETECT_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_INA233_E1S_BOOT_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_VR_CPU0_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_VR_CPU1_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_VR_PVDD11_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_VR_PVDDIO_VOLT_V
    │   ├─ /xyz/openbmc_project/sensors/voltage/Sentinel_Dome_Slot_1_MB_VR_SOC_VOLT_V
